### PR TITLE
test: XrpCurrencyAmount

### PR DESCRIPTION
**Pull request by mukulljangid**
Add tests for class XrpCurrencyAmount

### CODE REVIEW

**Comment by sappenin** [comment]
Fixes #73

File: xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/transactions/XrpCurrencyAmountTest.java [hunk 1]
```diff
@@ -64,4 +67,38 @@ public void toXrp() {
...
+
+    assertThat(
+      XrpCurrencyAmount.ofXrp(XrpCurrencyAmount.ofDrops(1L).toXrp())
+        .plus(XrpCurrencyAmount.ofXrp(XrpCurrencyAmount.ofDrops(0L).toXrp()))
+    ).isEqualTo(XrpCurrencyAmount.ofDrops(1L));
+  }
+
+  @Test
+  public void minusXrp() {
+    assertThat(
```
Comment (sappenin) [inline xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/transactions/XrpCurrencyAmountTest.java:85]: Can we add a unit test to cover the case where subtraction yields a negative number?

**Review by sappenin — CHANGES_REQUESTED** [review]

File: xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/transactions/XrpCurrencyAmountTest.java [hunk 2]
```diff
@@ -64,4 +67,38 @@ public void toXrp() {
     assertThat(XrpCurrencyAmount.ofDrops(MAX_XRP_IN_DROPS - 1).toXrp()).isEqualTo(new BigDecimal("99999999999.999999"));
   }
```
Comment (sappenin) [inline xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/transactions/XrpCurrencyAmountTest.java:68]: Let's also add coverage of the `toString()` method.

File: xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/transactions/Wrappers.java [hunk 3]
```diff
@@ -114,6 +114,7 @@ public int hashCode() {
   abstract static class _XrpCurrencyAmount extends Wrapper<UnsignedLong> implements Serializable, CurrencyAmount {
 
     static final long ONE_XRP_IN_DROPS = 1_000_000L;
+    static final long TWO_XRP_IN_DROPS = 2_000_000L;
```
Comment (nkramer44) [inline xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/transactions/Wrappers.java:117]: Can we move this to the test class if it's not used elsewhere?

File: xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/transactions/XrpCurrencyAmountTest.java [hunk 4]
```diff
@@ -64,4 +67,38 @@ public void toXrp() {
...
 
+  @Test
+  public void plusXrp() {
+    assertThat(
+      XrpCurrencyAmount.ofDrops(HALF_XRP_IN_DROPS)
+        .plus(XrpCurrencyAmount.ofDrops(HALF_XRP_IN_DROPS))
+    ).isEqualTo(XrpCurrencyAmount.ofDrops(ONE_XRP_IN_DROPS));
+
+    assertThat(
+      XrpCurrencyAmount.ofXrp(XrpCurrencyAmount.ofDrops(1L).toXrp())
```
Comment (nkramer44) [inline xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/transactions/XrpCurrencyAmountTest.java:78]: Couldn't you just do `XrpCurrencyAmount.ofDrops(1L)`?

**Review by nkramer44 — CHANGES_REQUESTED** [review]

**Review by nkramer44 — APPROVED** [review]

**Review by sappenin — APPROVED** [review]
